documentation/doxygen/Makefile

```diff
@@ -44,5 +49,5 @@ doxygen: filter
 	rm -f $(DOXYGEN_NOTEBOOK_PATH)/*.root
 
 clean:
-	rm -r $(DOXYGEN_OUTPUT_DIRECTORY) filter htmlfooter.html dataset*
+	rm -r $(DOXYGEN_OUTPUT_DIRECTORY) filter htmlfooter.html dataset* $(PYZ_DIR)/*.pyzdoc
```

Comment:
It's not nice that the `.pyzdoc` end up in the source folder. Wouldn't a temp folder in the doxygen directory be better? This could also make it easier to suppress them in `.gitignore`

Reply:
I thought it was fine to write temporaries in the source directory, since in that line a few others are cleaned too. Also, I need the `.pyzdoc` to be somewhere in the `DOXYGEN_SOURCE_DIRECTORY` to be processed by Doxygen, right?

Reply:
The ones that are cleaned live in
`documentation/doxygen`, the new ones in
`bindings/pyroot/pythonizations/python/ROOT/pythonization`, so they are kind of next to the source code.

In the nightly doxygen job, that's totally fine, because everything gets wiped, but if you run locally while also developing, it's a bit less convenient.

> Also, I need the .pyzdoc to be somewhere in the DOXYGEN_SOURCE_DIRECTORY to be processed by Doxygen, right?

Sure, but I thought that `documentation/doxygen/pyroot_tmp` or so could also work.

Reply:
I see your point, I think it's fine they are generated by the `.py`s, if you want to get rid of them while developing you can `make clean` them (which is what I did).